operator/v1/types_network.go

```diff
@@ -321,6 +321,11 @@ type OVNKubernetesConfig struct {
 	// not using OVN.
 	// +optional
 	HybridOverlayConfig *HybridOverlayConfig `json:"hybridOverlayConfig,omitempty"`
+	// enableIPsec enables IPsec encryption on all pod-pod traffic within the cluster.
```

Comment:
Do we need to make the distinction that this applies to only pods on the pod-network and host-network pods won't be using IPsec?

Reply:
It's a bit confusing to describe but I will paraphrase what you noted here.